pkg/brew/tome.rb

```diff
@@ -0,0 +1,24 @@
+class Tome < Formula
+  HOMEPAGE = "https://github.com/toumorokoshi/tome"
+
+  desc "Modern replacement for 'sub'"
+  homepage HOMEPAGE
+  url HOMEPAGE
+  license "MIT"
+  version '0.1.0'
+  revision 1
+
+  head "#{HOMEPAGE}.git"
+
+  depends_on "rust" => :build
```

Comment:
It might be a heavy dependency for some users to require rust to install the relatively small tome.

Is it acceptable to download a release instead?

Reply:
We can do that also. People want/expect the ability to compile if desired.

What we could do is setup homebrew to either get most recent release, OR compile from scratch if wanting it built from master.

If you like that approach, I'll break out a follow up task to address.

Reply:
Broke into new issue.